edb/pgsql/delta.py

```diff
@@ -2867,6 +2870,9 @@ def apply(
         prop = self.scls
         schema = PropertyMetaCommand.apply(self, schema, context)
 
+        if self.metadata_only:
```

Comment:
Perhaps we can generalize this by skipping `metadata_only` instances in `Adapter.adapt()` called by `compiler.py:_process_delta`?

Reply:
I'm not sure how to make that work; we do need to process those instances, just stop after applying the schema changes and handling children.

Reply:
The idea is that you don't "adapt" `metadata_only` commands into equivalent `pgsql.Command` instances. It's fine to leave this to a followup cleanup.